### ISSUE TITLE
Feature request: dbltap event

### ISSUE THREAD

**Issue by nolimitdev** [jerrybendy/vue-touch-events#64]
Event dbltab should be great alternative to native dblclick. Why? Because of native dblclick does not work on iOS. Double tab speed tolerance in ms would be fine to be configurable.

**Comment by jerrybendy** [comment]
Hi @nolimitdev ,

I don't think it's a good idea to support `dbltap`. Why we use an alternative `tap` event instead of `click`? On touch screen devices, when you tap on the screen, the browser doesn't know what you actually want to do, `click` or `dblclick`? So that the browser has to wait 300ms to ensure this is a `click` or a `dblclick`. This causes a bad user experience and the `tap` event is focused on solve this issue.

Back to your feature request, if we support `dbltap`, how could we resolve the conflict between `tap` and `dlbtap`?

**Comment by nolimitdev** [comment]
I will reply with question. On desktop browser with mouse how is natively solved difference between click and dblclick? It works well in practice. E.g tap can work normally without change but when there is only dbltab I think there is no problem when user do not use tap. When used both tab and dbltap, only in this case apply needed ms delay to ensure it is dbltap or tap.

**Comment by jerrybendy** [comment]
Yes you are right. I tried this on Chrome, and when bind `click` and `dblclick` at same time and double click it, it will trigger 2 `click`s and 1 `dblclick`. The best way to resolve this conflict is do not resolve it 😀

I will consider how to reliaze it. Thanks for your advice.

**Comment by nolimitdev** [comment]
I agree that no conflict resolving is needed. When user use just dbltab, no problem. In my case Im now using v-on:click and v-on:dblclick and each does something and I never mind that clicks are fired within one dblclick (it is desirable in my case). But dblclick is not supporteted on iOS and by googling users complains about it for long years and have to use different hacks to bypass it. It will be great vue-touch-events solve this shame of Apple.

Btw anyone can simple solve conflict v-on:click/v-on:dblclick or v-touch:tap/v-touch:dbltap by yourself... apply timeout before run click/tab callback, when fired dblclick/dbltap clear timeout and fire just dblclick/dbltap callback. But in my case firing all events is desirable.

**Comment by jerrybendy** [comment]
iOS uses dblclick to zoom in and zoom out. Maybe this is the reason why iOS doesn't support dblclick.

**Comment by euan-smith** [comment]
Came here looking for exactly the same feature - I'm implementing a pan/zoom control on an image and want to exactly reproduce the double-tap to zoom in/out which people generally expect, so in this case I will just ignore single tap events so no problem both being fired.  I'll work around it, but it would be really nice to have.

I do understand the problem with the delay - this is what the fast-click library was working around, but I think that delay is now gone from browsers (see [here](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away))  Just firing both and letting the page deal with it is definately the right solution.
